extension/chrome/elements/pgp_block.ts

```diff
@@ -436,6 +453,28 @@ Catch.try(async () => {
     }
   };
 
+  const addSecondPart = (message: string, isHtml: boolean = false) => {
+    const pgpBlock = $("#pgp_block")
+      .append('<div id="action_open_full" class="three_dots"><img src="/img/svgs/three-dots.svg" /></div>'); // xss-safe-factory
```

Comment:
Okay, will do. 

Is there an article or some documentation where I can read about these comments because I can't understand exactly which xss comment I should use and where?

Reply:
I'll make docs about it, please make an issue for me

Reply:
Yes, I've done it.
#1752